tasks/Ubuntu.yml

```diff
@@ -38,7 +40,7 @@
 
 - name: "Update APT cache"
   apt:
-    update_cache: yes
+    update_cache: "yes"
```

Comment:
It's not `"yes"` but `true`
`update_cache=dict(aliases=['update-cache'], type='bool'),`

Reply:
I only added the quotes, but OK I'll fix it